docs/other/troubleshooting-gratia.md

```diff
@@ -0,0 +1,375 @@
+
+
+4. If this has been a persistent problem, you might have many records. After a while, they are put into a compressed files in another directory. You can move those files, then uncompress them. This is a long name: note that the path ends in "staged/store" instead of "outbox" as above:
+
+        :::console
+        %RED%# Find the old files%ENDCOLOR%
+        root@host # cd /var/lib/gratia/tmp/gratiafiles/subdir.condor_fermicloud084.fnal.gov_%RED%aitarg%ENDCOLOR%-osg-itb.opensciencegrid.org_80/staged/store
+
+        %RED%# Move them to the correct directory%ENDCOLOR%
+        root@host # mv tz* /var/lib/gratia/tmp/gratiafiles/subdir.condor_fermicloud084.fnal.gov_%RED%gratia%ENDCOLOR%-osg-itb.opensciencegrid.org_80/outbox/.
+        root@host # cd !$
+
+        %RED%# For each tz file:%ENDCOLOR%
+        root@host # tar xf tz.1223.... [name shortened for legibility]
+        root@host # rm tz.1223....
+
+
+    When you've done this, you can re-run the Gratia probe by hand, or wait for it to run via cron.
+
+Appendix: Important Gratia files
+--------------------------------
+
+This document cannot cover all the errors you might experience. If you need to look for more data, you can look at log files for the various services on your CE.
+
+| File                                                | Purpose                                                                                                                                                                            |
+|:----------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `/var/log/gratia/%RED%DATE%ENDCOLOR%.log`           | Log file that records information about processing and uploading of Gratia accounting data                                                                                         |
+| `/var/log/gratia/gridftpTransfer.log`               | Log file specific to the Gratia GridFTP probe                                                                                                                                      |
+| `/var/lib/gratia/data`                              | Location for Condor and PBS job data before being processed by Gratia<br>Condor's `PER_JOB_HISTORY_DIR` should be set to this location                                       |
+| `/var/lib/gratia/tmp/gratiafiles`                   | Location for temporary Gratia data as it is being processed, usually empty.<br>If you have files that are more than 30 minutes old in this directory, there may be a problem |
+| `/etc/gratia/%RED%PROBE-NAME%ENDCOLOR%/ProbeConfig` | Configuration for Gratia probes, one per probe type</br>Normally you don't need to edit this                                                                                 |
+
+The most common RPMs you will see are:
```

Comment:
We should indicate that you will not see all of these on a single host

Reply:
Eh, most of them will be.  The only entry that will not be on every host clearly says "Log file specific to the Gratia GridFTP"

Reply:
I believe that I was referring to the RPMs below